src/test/java/org/sagebionetworks/web/unitserver/StackEndpointsTest.java

```diff
@@ -0,0 +1,157 @@
+package org.sagebionetworks.web.unitserver;
+
+import static org.junit.Assert.assertEquals;
+import static org.sagebionetworks.web.server.StackEndpoints.AUTH_ENDPOINT_KEY;
+import static org.sagebionetworks.web.server.StackEndpoints.AUTH_SUFFIX;
+import static org.sagebionetworks.web.server.StackEndpoints.FILE_ENDPOINT_KEY;
+import static org.sagebionetworks.web.server.StackEndpoints.FILE_SUFFIX;
+import static org.sagebionetworks.web.server.StackEndpoints.PARAM3;
+import static org.sagebionetworks.web.server.StackEndpoints.PARAM4;
+import static org.sagebionetworks.web.server.StackEndpoints.PARAM5;
+import static org.sagebionetworks.web.server.StackEndpoints.REPO_ENDPOINT_KEY;
+import static org.sagebionetworks.web.server.StackEndpoints.REPO_SUFFIX;
+import static org.sagebionetworks.web.server.StackEndpoints.STACK_BEANSTALK_NUMBER_PROPERTY_NAME;
+import static org.sagebionetworks.web.server.StackEndpoints.STACK_INSTANCE_PROPERTY_NAME;
+import static org.sagebionetworks.web.server.StackEndpoints.STACK_PROPERTY_NAME;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.sagebionetworks.web.server.StackEndpoints;
+
+public class StackEndpointsTest {
+
+  @Before
+  public void setup() {
+    clearProperties();
```

Comment:
Currently reverts the change before the test is executed, which is great.  But shouldn't this be after the test is executed so it does not have the potential to pollute any downstream tests?